aikau/src/test/resources/alfresco/forms/controls/MultiSelectInputTest.js

```diff
@@ -74,13 +77,17 @@ define([
             //    focused element: none
             //    
             "Controls are fully loaded": function() {
-               return browser.findAllByCssSelector("#MULTISELECT_1 .alfresco-forms-controls-MultiSelect--loaded")
+               // Get the selectors required for this test...
+               var selector1 = TestCommon.getTestSelector(MultiInputSelectSelectors, "options.loaded.state", ["MULTISELECT_1"]);
+               var selector2 = TestCommon.getTestSelector(MultiInputSelectSelectors, "options.loaded.state", ["MULTISELECT_2"]);
```

Comment:
It might be more readable (when you're halfway down a page of code) to have something more like:

```javascript
var selectors = {
   control1: TestCommon.getTestSelector(MultiInputSelectSelectors, "options.loaded.state", ["MULTISELECT_1"]),
   control2: TestCommon.getTestSelector(MultiInputSelectSelectors, "options.loaded.state", ["MULTISELECT_2"]),
};
```

Then you could just refer to this object in the code (e.g. `selector.control1`), rather than knowing which `selector1` is, or indeed `selector23`.